components/common/Button/button.scss

```diff
@@ -17,35 +16,80 @@
 	}
 
 	&--primary {
-		background-color: $button-primary-background-color;
-		color: $button-text-color;
+		background-color: $blue-light;
+		color: $button-text-light;
 	}
 
 	&--secondary {
-		color: $button-secondary-text-color;
+		color: $blue-light;
 		background-color: transparent;
 	}
 
+	&--gray,
+	&--transparent {
+		color: $text-lighter;
+		font-weight: 400;
+	}
+
+	&--transparent,
+	&--red {
+		background-color: transparent;
+	}
```

Comment:
red na pewno nie powinno być transparent :D dlatego jak wspomniałam to nazewnistwo klas nie powinno tak wyglądać